src/phases/faint-phase.ts

```diff
@@ -189,7 +188,7 @@ export class FaintPhase extends PokemonPhase {
       /** The total number of legal player Pokemon that aren't currently on the field */
       const legalPlayerPartyPokemon = legalPlayerPokemon.filter((p) => !p.isActive(true));
       if (!legalPlayerPokemon.length) {
-        globalScene.unshiftPhase(new GameOverPhase());
+        globalScene.gameOver({ clearPhaseQueue: true });
```

Comment:
```suggestion
        globalScene.gameOver({ clearPhaseQueue: false });
```
it doesn't seem like the phase queue was being cleared before here?